Mention that `ttl_delete_rate_limit` is per node

rmloveland opened the pull request:
... and show how to calculate the rate limit using `SHOW RANGES`

Fixes 

+ DOC-8877
+ DOC-9059
+ DOC-9358

rafiss reviewed (approved):
lgtm!

not sure if you prefer to do a separate PR for this, but i also notice this table is missing `ttl_select_rate_limit`, which works similarly. cc @dikshant 

see: https://cockroachlabs.atlassian.net/issues/?jql=project%20%3D%20DOC%20AND%20text%20~%20%22ttljob%20ttl_select_rate_limit%22

rmloveland commented:
> lgtm!

SWEET!

> not sure if you prefer to do a separate PR for this, but i also notice this table is missing `ttl_select_rate_limit`, which works similarly. cc @dikshant

thank you! did not know about that one

re: getting it into this PR, I saw Dikshant's other comment that we should add it here, so I'll update this PR
 
> see: https://cockroachlabs.atlassian.net/issues/?jql=project%20%3D%20DOC%20AND%20text%20~%20%22ttljob%20ttl_select_rate_limit%22

based on looking at those, it looks like `til_select_rate_limit` is only in v23.2+, is that correct?

rmloveland commented:
@rafiss and @dikshant

thanks for your feedback!

in the latest changes I've added some docs for `ttl_select_rate_limit`, and also ported the changes to the description of `til_delete_rate_limit` to v22.2, v23.1, and v23.2 docs

the descriptions of how `ttl_select_rate_limit` and `ttl_delete_rate_limit` are also changed to use (almost) the exact same language for slightly easier maintenance

PTAL!

rmloveland commented:
@rafiss I know you already LGTM'd but do you mind giving the latest update a quick look?

not much changed of substance, just wanted to make sure the updates re: `til_select_rate_limit` are accurate before sending this on for docs team review

rafiss reviewed (approved):
sorry for missing this. lgtm!

dikshant reviewed (approved):
LGTM!

taroface commented on src/current/v22.2/row-level-ttl.md:
```diff
@@ -140,7 +140,7 @@ The settings that control the behavior of Row-Level TTL are provided using [stor
 | `ttl_select_batch_size`                                                    | How many rows to [select](select-clause.html) at one time during the row expiration check. Default: 500. Minimum: 1.                                                                                                                                                                                                             ...
 | `ttl_delete_batch_size` <a name="param-ttl-delete-batch-size"></a>         | How many rows to [delete](delete.html) at a time. Default: 100. Minimum: 1.                                                                                                                                                                                                                                                      ...
-| `ttl_delete_rate_limit`                                                    | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit).                                                                                                                                                                                                                                             ...
+| `ttl_delete_rate_limit` | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit). <br/><br/> Note: The rate limit will vary in practice based on the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) and [leaseholders]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder). It is ap ...
```
```suggestion
| `ttl_delete_rate_limit` | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit). <br/><br/> Note: The rate limit is applied per leaseholder per table. In practice, it will vary based on the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) and [leaseholders]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder).<br/> For example, if this variable is set to 500, and a table `public.foo` has its ranges spread across 3 leaseholders, then the cluster-wide rate limit for `public.foo` is 500 x 3 = 1500. | `sql.ttl.default_delete_rate_limit` |
```

rmloveland replied:
thanks, this is way better.  applied to all versions of this text

taroface commented on src/current/v23.1/row-level-ttl.md:
```diff
@@ -140,7 +140,7 @@ The settings that control the behavior of Row-Level TTL are provided using [stor
 | `ttl_select_batch_size`                                                    | How many rows to [select]({% link {{ page.version.version }}/select-clause.md %}) at one time during the row expiration check. Default: 500. Minimum: 1.                                                                                                                                                                         ...
 | `ttl_delete_batch_size` <a name="param-ttl-delete-batch-size"></a>                                                    | How many rows to [delete]({% link {{ page.version.version }}/delete.md %}) at a time. Default: 100. Minimum: 1.                                                                                                                                                                       ...
-| `ttl_delete_rate_limit`                                                    | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit).                                                                                                                                                                                                                                             ...
+| `ttl_delete_rate_limit` | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit). <br/><br/> Note: The rate limit will vary in practice based on the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) and [leaseholders]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder). It is ap ...
```
Same suggestion as earlier

rmloveland replied:
✅

taroface commented on src/current/v23.2/row-level-ttl.md:
```diff
@@ -142,7 +142,8 @@ The settings that control the behavior of Row-Level TTL are provided using [stor
 | `ttl_select_batch_size`                                                    | How many rows to [select]({% link {{ page.version.version }}/select-clause.md %}) at one time during the row expiration check. Default: 500. Minimum: 1.                                                                                                                                                                         ...
 | `ttl_delete_batch_size` <a name="param-ttl-delete-batch-size"></a>                                                    | How many rows to [delete]({% link {{ page.version.version }}/delete.md %}) at a time. Default: 100. Minimum: 1.                                                                                                                                                                       ...
-| `ttl_delete_rate_limit`                                                    | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit).                                                                                                                                                                                                                                             ...
+| `ttl_select_rate_limit` | Maximum number of rows to be selected per second (rate limit). Default: 0 (no limit). <br/><br/> Note: The rate limit will vary in practice based on the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) and [leaseholders]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder). It is a ...
```
Same suggestion as earlier

taroface commented on src/current/v23.2/row-level-ttl.md:
```diff
@@ -142,7 +142,8 @@ The settings that control the behavior of Row-Level TTL are provided using [stor
 | `ttl_delete_batch_size` <a name="param-ttl-delete-batch-size"></a>                                                    | How many rows to [delete]({% link {{ page.version.version }}/delete.md %}) at a time. Default: 100. Minimum: 1.                                                                                                                                                                       ...
-| `ttl_delete_rate_limit`                                                    | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit).                                                                                                                                                                                                                                             ...
+| `ttl_select_rate_limit` | Maximum number of rows to be selected per second (rate limit). Default: 0 (no limit). <br/><br/> Note: The rate limit will vary in practice based on the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) and [leaseholders]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder). It is a ...
+| `ttl_delete_rate_limit` | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit). <br/><br/> Note: The rate limit will vary in practice based on the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) and [leaseholders]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder). It is ap ...
```
Same suggestion as earlier

taroface reviewed (approved):
LGTM; I have one suggestion that I think makes this easier to understand.
